docs/adr/0006-monitoring-structure.md

```diff
@@ -0,0 +1,55 @@
+# Application Monitoring in Operate First using Prometheus
+
+* Status: accepted
+* Deciders: 4n4nd, anishasthana, hemajv, HumairAK, tumido
+* Date: 2021-01-06
+
+Technical Story: [issue-1](https://github.com/operate-first/blueprint/issues/8)
+                 [issue-2](https://github.com/operate-first/apps/issues/57)
+
+## Context and Problem Statement
```

Comment:
I think we could use more information here to better understand the rest of the doc. For example I think some questions that might be worth answering:

- Why are we considering giving service accounts cross name permissions? Why aren't we using operatorgroups to do this? 
- Are we using ODH monitoring? 
- Are we using ODH to monitor non ODH services? 
- Why are we weighing options between multiple prometheus instances vs one

Basically, when I read the `Considered Options` below, I'm not sure why one ought to pick Option 1 over Option 2, because I don't know what our concerns/priorities are, etc.

Reply:
added some more context